javascript/challenges/fizzBuzzTree/buzz-tree.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+class Node {
+  constructor(value, left=null, right=null) {
+    this.value = value;
+    this.left = left;
+    this.right = right;
+  }
+}
+
```

Comment:
The biggest issue that I am finding is that root is coming back as undefined. Some ways to solve that is to bring that in from your tree file or you can instantiate one here